internal/lib/stats/stats_server_linux.go

```diff
@@ -68,14 +67,20 @@ func (ss *StatsServer) updateSandbox(sb *sandbox.Sandbox) *types.PodSandboxStats
 			continue
 		}
 
-		cgstats, err := ss.Runtime().ContainerStats(ss.ctx, c, sb.CgroupParent())
+		ctrStats, err := ss.Runtime().ContainerStats(ss.ctx, c, sb.CgroupParent())
 		if err != nil {
 			log.Errorf(ss.ctx, "Error getting container stats %s: %v", c.ID(), err)
 
 			continue
 		}
-		// Convert cgroups stats to CRI stats.
-		cStats := containerCRIStats(cgstats, c, cgstats.SystemNano)
+		diskStats, err := ss.Runtime().DiskStats(ss.ctx, c, sb.CgroupParent())
+		if err != nil {
+			log.Errorf(ss.ctx, "Error getting disk stats %s: %v", c.ID(), err)
+
+			continue
```

Comment:
I think we should drop the continues in either case and only continue if both diskSTats and ctrStats are nil